packages/astro/src/@types/astro.ts

```diff
@@ -2687,29 +2683,29 @@ export type SSRComponentMetadata = {
 };
 
 export interface SSRResult {
+	/**
```

Comment:
The move around of properties makes it difficult to understand what actually changed. We removed only `cookies` did we? Can you explain why those properties were removed?

Reply:
Yes, only cookies were removed - nothing used them anymore. The others were moved to keep them alphabetical - it keeps them scannable and is the general pattern of most interfaces.